generator/ServiceClientGeneratorLib/Member.cs

```diff
@@ -757,7 +764,7 @@ public string DetermineTypeUnmarshallerInstantiate(JsonData extendedData)
                 case "list":
                     var listType = DetermineType(memberShape[Shape.MemberKey], true);
                     var listTypeUnmarshaller = GetTypeUnmarshallerName(memberShape[Shape.MemberKey]);
-                    var listTypeUnmarshallerInstantiate = DetermineTypeUnmarshallerInstantiate(memberShape[Shape.MemberKey]);
+                    var listTypeUnmarshallerInstantiate = DetermineTypeUnmarshallerInstantiate(memberShape[Shape.MemberKey], typeNode.ToString());
```

Comment:
Issue 28 is about unmarshalling flattened maps, but I see that you are tweaking lists as well. Is this just a case of "I might as well fix this while I'm here" or does this affect the map generation somehow?

Reply:
This was passing in the parent type since I had it to call the appropriate method. I suppose it could have called the override without the type node but since I had it I called the correct method. It won't do anything with parent type node of list at this time.

Reply:
ah okay makes sense